core/src/main/scala/org/http4s/websocket/Websocket.scala

```diff
@@ -4,6 +4,9 @@ import fs2._
 import org.http4s.websocket.WebsocketBits.WebSocketFrame
 
 private[http4s] final case class Websocket[F[_]](
-    read: Stream[F, WebSocketFrame],
-    write: Sink[F, WebSocketFrame]
-)
+    @deprecatedName('read, "0.18-M6") send: Stream[F, WebSocketFrame],
+    @deprecatedName('write, "0.18-M6") receive: Sink[F, WebSocketFrame]
```

Comment:
Sorry, one last nitpick: the version that will include this is 0.18.0-M7.

Reply:
Whoop, right :+1:  Thank You